### PR TITLE
Bus leds

### PR DESCRIPTION
**Pull request by domiko96**
Please this time @aewatt95

### CODE REVIEW

File: software/platformio.ini [hunk 1]
```diff
@@ -1,32 +0,0 @@
-; PlatformIO Project Configuration File
```
Comment (aewatt95) [inline software/platformio.ini]: Please note that this file is FUCKING IMPORTANT!!!

**Review by aewatt95 — CHANGES_REQUESTED** [review]
You deleted the platformio.ini file. If i were you, i would reconsider this move.

File: software/platformio.ini (hunk 1 above)
Comment (domiko96) [reply software/platformio.ini]: Oh dear your right bra

**Review by aewatt95 — APPROVED** [review]
NICE!

**Pull request merged**